packages/wallet/backend/src/rafiki/service.ts

```diff
@@ -126,7 +234,8 @@ export class RafikiService implements IRafikiService {
     if (
       [
         EventType.OutgoingPaymentCreated,
-        EventType.OutgoingPaymentCompleted
+        EventType.OutgoingPaymentCompleted,
+        EventType.OutgoingPaymentFailed
```

Comment:
Bug fix here: We have to parse `debitAmount` even for  `OutgoingPaymentFailed` event otherwise it throws a bad request  error - few lines below - and the event is not handled at all